pepper-apis/src/main/java/org/broadinstitute/ddp/event/pubsubtask/impl/updateprofile/UpdateProfileProcessor.java

```diff
@@ -52,6 +52,6 @@ protected void updateData(PubSubTask pubSubTask) {
 
         new UpdateEmailHandler().updateEmail(participantGuid, properties);
```

Comment:
May be we can merge EmailHandler into UpdateProfileDataHandler !
It does some additional work like updating Auth0 though. Should be ok to leave it for now